yt_dlp/extractor/rozhlas.py

```diff
@@ -45,3 +47,70 @@ def _real_extract(self, url):
             'duration': duration,
             'vcodec': 'none',
         }
+
+
+class RozhlasVltavaIE(InfoExtractor):
+    _VALID_URL = r'https?://((vltava|wave|radiozurnal|dvojka|plus|sport|d-dur|jazz|junior|pohoda)\.rozhlas|english\.radio)\.cz/[a-z|-]*-(?P<id>\d+)'
+    _TESTS = [{
+        'url': 'https://wave.rozhlas.cz/papej-masicko-porcujeme-a-bilancujeme-filmy-a-serialy-ktere-letos-zabily-8891337',
+        'md5': 'ba2fdbc1242fc16771c7695d271ec355',
+        'info_dict': {
+            'id': '8891337',
+            'ext': 'mp3',
+            'title': 'Máte po celé sezóně poslouchání Čelistí právo na wellness, nebo můžete být rádi, že jste vůbec naživu? Poslechněte si filmově kritickou bilanci všeho, co v uplynulých měsících stálo za vidění',
+            'description': '8888928: Vánoce s Radiem Wave | 7702992: Český rozhlas má nadílku pro každého',
+            'duration': 1574
+        }
+    }]
+
+    def find_element(self, webpage):
+        # Use utils.get_element_text_and_html_by_tag() instead when it accepts less strict html.
+
+        playerDiv = ''
+        for k, line in enumerate(webpage.split("\n")):
+            if line.find('mujRozhlasPlayer') != -1:
+                playerDiv = line.strip()
+                break
+
+        if playerDiv.count('<div') > 1:
+            for k, element in enumerate(playerDiv.split('<div')):
+                if element.count('mujRozhlasPlayer') == 1:
+                    playerDiv = '<div' + element
+                    break
+
+        return playerDiv
+
+    def _real_extract(self, url):
+        webpage = self._download_webpage(url, None)
+
+        playerDiv = self.find_element(webpage)
+
+        jsonString = extract_attributes(playerDiv).get('data-player')
+        jsonData = json.loads(jsonString)
+
+        entries = []
+        for entry in jsonData.get("data").get('playlist'):
+            entry0 = entry.get('audioLinks')[0]
```

Comment:
why only 0?

Reply:
They support just one format (mp3). I didn't find anything I can test it on.